lib/shim/transaction-shim.js

```diff
@@ -218,7 +218,7 @@ function setTransactionName(name) {
  * @param {string} [transportType='Unknown']
  *  The transport type that brought the headers. Usually `HTTP` or `HTTPS`.
  */
-function handleCATHeaders(headers, segment, transportType) {
+function handleMqTracingHeaders(headers, segment, transportType) {
   // TODO: rename function or replace functionality when CAT fully removed.
```

Comment:
I would update this comment to only include `replace functionality when CAT is full removed` as you already renamed.  I'm also wondering if this should just move to the message-shim as it's already breaking because of the name and it's the only shim that is using it

Reply:
I can take care of this and then merge